tests/test_helper/lxd.bash

```diff
@@ -0,0 +1,40 @@
+LXC_TEST_IMAGE=jammy
+DEPENDENCIES="jq"
+
+function start_containers() {
```

Comment:
The functions in this file can potentially be used with multiple backends which we could possibly implement in other modules, for the LXD backend it can also be used for both containers and virtual machines.

I wonder if we should use a more generic name, such as `instances`, so that we can use it for both? We then have an opportunity to run the same tests using containers and virtual machines by modifying an environment variable or something similar.